app/controllers/IndexController.scala

```diff
@@ -74,15 +78,17 @@ trait IndexController extends TaxsController {
         } map {
           x => Redirect(routes.IndexController.authorisedIndex()).withSession(session)
         }
+      }
       case _ => {
         show(user, request)
       }
     }
 
   type T = AtsList
```

Comment:
can we change this, I know its not technically the scope of this ticket... but its awful

Reply:
type changed to be more specific